src/main/ui/chat/chat-appearance.gd

```diff
@@ -38,10 +37,9 @@ See ChatEvent.accent_def for a full description of the accent_def properties.
 func _init(accent_def: Dictionary):
 	accent_color = accent_def["color"] if accent_def.has("color") else Color.gray
 	accent_scale = accent_def["accent_scale"] if accent_def.has("accent_scale") else 2.0
-	accent_swapped = accent_def["accent_swapped"] if accent_def.has("accent_swapped") else false
+	accent_swapped = true if accent_def.has("accent_swapped") else false
```

Comment:
`true if accent_def.get`... not has. It could have false.

Reply:
Simplified to use map.get(key) function. Simplified all other code here to use map.get(key) or map.get(key, default), none of them needed the ternary operator.